core/server/utils/image-size.js

```diff
@@ -143,6 +143,9 @@ getImageSizeFromUrl = function getImageSizeFromUrl(imagePath) {
             context: err.url || imagePath
         }));
     }).catch(function (err) {
+        if (err instanceof errors.NotFoundError || err instanceof errors.InternalServerError) {
```

Comment:
This can be simplified to `err instanceof errors.GhostError` I think - cc @kirrg001 on the proper way to detect pre-handled errors (we already do this here though: https://github.com/TryGhost/Ghost/blob/master/core/server/helpers/register.js#L20)

We should trust that if the error has already been handled it was handled correctly.

Reply:
Oops.... I thought I tried that at first and it didn't work, but I didn't do it right. Pushed the changes.